easybuild/easyconfigs/n/Nipype/Nipype-1.2.3-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,149 @@
+easyblock = 'PythonBundle'
+
+name = 'Nipype'
+version = '1.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://nipy.org/nipype'
+description = """Nipype is a Python project that provides a uniform interface to existing neuroimaging software and
+ facilitates interaction between these packages within a single workflow."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('lxml', '4.2.5', versionsuffix),
+    ('NiBabel', '2.5.1', versionsuffix),
+]
+
+# raise an error when auto-downloading of dependency packages is detected
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_defaultclass = 'PythonPackage'
```

Comment:
now that this is a `PythonBundle`, `exts_default_options` and `exts_defaultclass` are no longer necessary and `use_pip: True` should be outside `exts_default_options`?